test/getCoverageLIb.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { describe, it } from "node:test";
+import assert from "node:assert";
+import { getCoverageLib } from "../src/utils";
```

Comment:
```suggestion
// Import Node.js Dependencies
import { describe, it } from "node:test";
import assert from "node:assert";

// Import Internal Dependencies
import { getCoverageLib } from "../src/utils";
```

Same for the other file